bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/rechnung/InvoiceListView.java

```diff
@@ -119,25 +127,33 @@ public void run() {
 
 		@Override
 		public void run() {
-			String idList = ConfigServiceHolder.get().get(ContextServiceHolder.get().getActiveUserContact().get(),
-					CFG_MANDATORFILTER, StringUtils.EMPTY);
-			MandantSelectorDialog dialog = new MandantSelectorDialog(getSite().getShell(), idList);
+			GenericSearchSelectionDialog dialog = new GenericSearchSelectionDialog(getSite().getShell(),
+					coreModelService.getQuery(IUser.class)
+							.and(ModelPackage.Literals.IMANDATOR__ACTIVE, COMPARATOR.EQUALS, true).execute(),
+					Messages.Core_Select_Mandator, Messages.Core_Select_Mandator, Messages.Core_Select_Mandator_Tooltip,
+					null, SWT.CHECK);
```

Comment:
Bitte hier wieder den MandantSelectorDialog verwenden, da wurde in #749 angepasst das nur aktive angezeigt werden.

Reply:
Soll ich meine Änderung rückgängig machen? Hier geht es nicht nur darum, dass nur die aktiven angezeigt werden, sondern auch um die Darstellung allgemein.